quidel_covidtest/delphi_quidel_covidtest/pull.py

```diff
@@ -110,11 +110,11 @@ def fix_date(df, logger):
     df.insert(2, "timestamp", df["TestDate"])
 
     mask = df["TestDate"] <= df["StorageDate"]
-    logger.info(f"Removing {((len(df) - np.sum(mask)) * 100 / len(df)):.2f}% of unusual data")
+    logger.info("Removing unusual data", percent=((len(df) - np.sum(mask)) * 100 / len(df)))
```

Comment:
The original had the `:.2f` which limits the decimal precision to 2 decimals, might be worth matching that.